specification/src/specification/queries/filtering.md

```diff
@@ -93,24 +70,11 @@ Binary comparison operators are denoted by expressions with a `type` field of `b
 
 See type [`ComparisonValue`](../../reference/types.md#comparisonvalue) for the valid inhabitants of the `value` field.
 
+_Note_: in general, `PathElement`s in a `ComparisonValue` can refer to _array_ relationships. However, in the case of the `in` operator, such requests can be very difficult to implement, and for practical purposes, it is not very useful to express such queries. Therefore, in the case of the `in` operator, connectors can expect `PathElements` in `ComparisonValue`s to always only refer to _object_ relationships, and fail with a `Bad Request` error otherwise.
```

Comment:
> can expect `PathElements` in `ComparisonValue`s to always only refer to _object_ relationships, and fail with a `Bad Request` error otherwise.

Note: assuming the phrasing here is intentional, and connectors are _not_ required to fail if this happens.
If connectors are required to fail we could phrase this strongly:

> can expect `PathElements` in `ComparisonValue`s to always only refer to _object_ relationships, and **must** fail with a `Bad Request` error otherwise.

Reply:
Right, I left it like this so that we could put it behind a capability if we decide it's useful later.

Reply:
Note though, there's no loss of expressivity, since `IN` is just an `or` expression in disguise anyway.